src/control-plane/backend/lambda/api/store/aws/quicksight.ts

```diff
@@ -87,8 +101,25 @@ export const registerQuickSightUser = async (email: string, username?: string) =
 
 export const registerClickstreamUser = async () => {
   const identityRegion = await sdkClient.QuickSightIdentityRegion();
-  await registerEmbeddingUserByRegion(identityRegion);
-  await registerQuickSightUserByRegion(identityRegion, QUICKSIGHT_EMBED_NO_REPLY_EMAIL, QUICKSIGHT_DASHBOARD_USER_NAME);
+  await registerUserByRegion(identityRegion, {
+    IdentityType: IdentityType.IAM,
+    AwsAccountId: awsAccountId,
+    Email: QUICKSIGHT_EMBED_NO_REPLY_EMAIL,
+    IamArn: QuickSightEmbedRoleArn,
+    Namespace: QUICKSIGHT_NAMESPACE,
+    UserRole: UserRole.ADMIN,
```

Comment:
Do we have to assign it to `ADMIN`? How about treating it as a `READER`?

Reply:
ExploreUser need Admin to create datasets.